### DISCUSSION TITLE
How do I get this to work in the dashboard?

### DISCUSSION THREAD

**Discussion by weirdyang** [AnderssonPeter/Hangfire.Console.Extensions#21] [Q&A] [answered]
I've added the extensions:

```
                builder.Services.AddSerilog((services, lc) => lc
                  .WriteTo.Hangfire()
                  .Enrich.WithHangfireContext()
                  .WriteTo.Console(theme: AnsiConsoleTheme.Code));

            builder.Services.AddHangfireConsoleExtensions();
```

but I still see nothing despite logging with ILogger

![image](https://github.com/user-attachments/assets/11bfacc7-12e6-4940-a39a-7744b0109b77)

**Comment by AnderssonPeter** [comment, marked as answer]
If you check out the repo and run the example projects, does it work then?

**Reply by weirdyang** [reply]
Thanks I realised I didn't call UseConsole on UseHangFire
